pkg/authorization/api/types.go

```diff
@@ -160,7 +160,7 @@ type Policy struct {
 	LastModified util.Time
 
 	// Roles holds all the Roles held by this Policy, mapped by Role.Name
-	Roles map[string]Role
+	Roles map[string]*Role
```

Comment:
Still looking at the other two causes for reflection.  

----- Original Message -----
> > @@ -160,7 +160,7 @@ type Policy struct {
> >  	LastModified util.Time
> >  
> >  	// Roles holds all the Roles held by this Policy, mapped by Role.Name
> > -	Roles map[string]Role
> > +	Roles map[string]*Role
> 
> so the external types are structs and the internal types are pointers?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/3269/files#r32655496